Runtime/Scripts/MobiledgeXIntegration.cs

```diff
@@ -302,50 +253,11 @@ public async Task<String> GetURI(LProto protocol = LProto.L_PROTO_TCP, int port
                 }
             }
             return uri;
-        }
-
-        /// <summary>
-        /// Wether Edge is Enabled on the device or not, Edge requires connections to run over cellular interface
-        /// </summary>
-        /// <returns> boolean value </returns>
-        public bool IsEdgeEnabled()
```

Comment:
Removing IsEdgeEnabled entirely?

Reply:
I think we still need it, even if not called (it's used, should be used) only just after FindCloudlet().

Reply:
Will be added later